docs/v1/openapi.yaml

```diff
@@ -1191,6 +1191,9 @@ components:
         tx_id:
           type: string
           description: To be used by Issuer applications for correlation if needed.
+        otp_pin:
+          type: string
+          description: Pre-authorized flow. Generated OTP pin for issuance. Issuer is responsible to deliver that pin to the final client (email,sms,etc).
```

Comment:
no need to mention below, explained well in spec 

```Issuer is responsible to deliver that pin to the final client (email,sms,etc)```

rename it from OTP to user_pin matching OIDC spec.